### PR TITLE
Error strings

### PR DESCRIPTION
**Pull request by kisom**
Prepend the package name to error strings.

### CODE REVIEW

File: examples/login/login.go [hunk 1]
```diff
@@ -4,12 +4,22 @@ import (
 	"encoding/hex"
 	"flag"
 	"fmt"
-	"github.com/conformal/yubikey"
+	"github.com/kisom/yubikey"
```
Comment (dgryski) [inline examples/login/login.go]: import path updated again

**Comment by kisom** [comment]
dgryski -- c2f1207fd64d5f1effa56f38a567687cd9878a54 fixes this after I talked to dajohi about it.

**Comment by dgryski** [comment]
The error_strings branch seems to have all the other fixes for ParseOtpString in it -- merging both will create conflicts (I think).

**Comment by kisom** [comment]
It looks like this pullreq pulled in stuff from another branch I had (that is in a separate pull request); c2f1207 and 8360453 should be in https://github.com/conformal/yubikey/pull/15

**Comment by dgryski** [comment]
When I work on third-party Go code, I leave it in $GOROOT with the 'origin' set to the proper upstream, but then add 'github.com/dgryski/foo' as a remote so that my code changes are living in github.com/someuser/foo even though I'm pushing somewhere else.  That way I don't have to update any of the import lines.

**Comment by dgryski** [comment]
This pull request has the commits from the other one because the branch you created was based off it instead of master.

(So, git checkout -b new-branch-name while you you were old-branch-name instead of trunk).  A rebranch and push -f  should fix it.

**Comment by dajohi** [comment]
implemented.

**Pull request closed without merging**